app/main/views/suppliers.py

```diff
@@ -72,18 +73,14 @@ def list_suppliers():
     suppliers = suppliers.distinct(Supplier.name, Supplier.supplier_id)
 
     try:
-        suppliers = suppliers.paginate(
+        return paginated_result_response(
+            result_name="suppliers",
```

Comment:
Given this set of views is primarily about suppliers, I think we _could_ define the resource name as suppliers and use that here. Anything else could continue to use string literals (eg contact information).

Can take it or leave it with this one, but just a thought.

Reply:
I've done as you suggested